secure/storage/src/storage.rs

```diff
@@ -21,3 +21,9 @@ pub enum Storage {
     NamespacedStorage(NamespacedStorage),
     OnDiskStorage(OnDiskStorage),
 }
+
+#[enum_dispatch(KVStorage, CryptoStorage)]
+pub enum StorageVariant {
+    Storage(Storage),
+    CachedStorage(CachedStorage<Storage>),
```

Comment:
Is there a reason for not including CachedStorage in the enum for Storage? much like NamespacedStorage?


Other than that, this is looking good.

Reply:
I didn't think of that! Okay so I just updated it and removed `StorageVariant`, thanks!